pom.xml

```diff
@@ -57,6 +57,24 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
yes, we can , I will remove it . I would suggest to leave the version update of maven-gpg-plugin